### DISCUSSION TITLE
Importing Issue (MMI v0.7.0 and RenderDoc v1.31)

### DISCUSSION THREAD

**Discussion by nghiasnuk** [eliemichel/MapsModelsImporter#348] [Q&A]
![20241130-205107](https://github.com/user-attachments/assets/66eaabbc-daad-4902-90c2-55e07611d13b)

Can you help me :<

**Comment by ull7i** [comment]
try RenderDoc V1.25, Blender V4.1, MMI V0.6.0

**Comment by chiisanthi** [comment]
I have the same issue, with blender 4.3.1 and renderdoc1.31, have you tried with older versions and if so, did it work?
